tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -368,6 +372,11 @@ public bool DiagnosticSourceEnabled
         /// </summary>
         internal DirectLogSubmissionSettings LogSubmissionSettings { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating the trace methods configuration.
+        /// </summary>
+        internal string TraceMethods { get; set; }
```

Comment:
Should we add it to telemetry? I guess it would be nice to see what customers are instrumenting, especially if 3rd party code

Reply:
Yes definitely! Added telemetry in 08e0eb8ca7aafbe0754f3b4c253b8c4fc002476a